packages/server/main.js

```diff
@@ -6,10 +6,74 @@ import { format, parse } from "json-rpc-protocol";
 // see https://koajs.com/
 const app = new Koa();
 
+var entries = new Map();
+let idCounter = 0;
+
 const METHODS = {
   subtract([a, b]) {
     return a - b;
   },
+
+  createEntry([nameEntry, contentEntry]) {
+    const entry = {
+      id: idCounter,
+      name: nameEntry,
+      content: contentEntry,
+      created: Date.now(),
+      updated: Date.now(),
+    };
+
+    idCounter += 1;
+    entries.set(entry.id, entry);
+
+    return entry.name + " created";
+  },
+
+  listEntries() {
+    let res = "";
+    entries.forEach(function(entry) {
+      res +=
+        "(" +
+        entry.id +
+        ") " +
+        entry.name +
+        " " +
+        entry.content +
+        " " +
+        entry.created +
+        " " +
+        entry.updated +
+        "\n";
+    });
+    return res;
```

Comment:
Simply returns a list of all entries, because a string is hard to use on client's side.